### PR TITLE
[SDPA-3841] ensure sass variables can be overridden

### PR DESCRIPTION
**Pull request by rubyllamadora**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-3841

### Changed

1.  Added !default rule to all color related styling so it can be overridden

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -20207,9 +20207,9 @@ websocket-driver@>=0.5.1:
     websocket-extensions ">=0.1.1"
 
 websocket-extensions@>=0.1.1:
-  version "0.1.3"
-  resolved "https://registry.yarnpkg.com/websocket-extensions/-/websocket-extensions-0.1.3.tgz#5d2ff22977003ec687a4b87073dfbbac146ccf29"
-  integrity sha512-nqHUnMXmBzT0w570r2JpJxfiSD1IzoI+HGVdd3aZ0yNi3ngvQ4jv1dtHt5VGxfI2yj5yqImPhOK4vmIh2xMbGg==
+  version "0.1.4"
+  resolved "https://registry.yarnpkg.com/websocket-extensions/-/websocket-extensions-0.1.4.tgz#7f8473bc839dfd87608adb95d7eb075211578a42"
+  integrity sha512-OqedPIGOfsDlo31UNwYbCFMSaO9m9G/0faIHj5/dZFDMFqPTcx6UwqyOy3COEaEOg/9VsGIpdqn62W5KhoKSpg==
```
Comment (rubyllamadora) [inline yarn.lock:20212]: Interesting.. I don't see this as one of the files changed when I do `git diff` nor in my commit list

**Comment by rubyllamadora** [comment]
> @rubyllamadora So all colours have been covered?

Yep. There are other styling that doesn't have !default but not colour related.

**Comment by tim-yao** [comment]
@rubyllamadora No need QA for this, please merge it directly. Thanks!

**Comment by rubyllamadora** [comment]
> @rubyllamadora No need QA for this, please merge it directly. Thanks!

Okay 👍. Will merge once cr approved by all.

**Pull request merged**